Docs/sphinx_documentation/source/GPU_Chapter.rst

```diff
@@ -8,8 +8,8 @@ NVIDIA, AMD and Intel GPUs using their native vendor language and therefore
 requires CUDA, HIP/ROCm and DPC++/SYCL, for NVIDIA, AMD and Intel GPUs, respectively.
 Users can also use OpenMP and/or OpenACC in their applications.
 
-AMReX supports NVIDIA GPUs with compute capability >= 6 and CUDA >= 10
-as well as CUDA 9.1.  While HIP and DPC++ compilers are in development in
+AMReX supports NVIDIA GPUs with compute capability >= 6 and CUDA >= 10.
```

Comment:
Is the statement on compute capability here consistent with the update to the CI?

Reply:
I changed the compute capability in the CI not because 6 didn't not work.  It was because we were not testing 7.